kurl-installer.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   contour:
     version: latest
-  docker:
+  containerd:
```

Comment:
lgtm

Reply:
Can you hit Approve under `Files Changed` > `Review Changes`?
Merging is blocked as 1 approving review is required